internal/ingress/controller/template/template.go

```diff
@@ -323,23 +323,28 @@ func buildProxyPass(host string, b interface{}, loc interface{}) string {
 	path := location.Path
 	proto := "http"
 
-	upstreamName := location.Backend
-	for _, backend := range backends {
-		if backend.Name == location.Backend {
-			if backend.Secure || backend.SSLPassthrough {
-				proto = "https"
-			}
+	upstreamName := "upstream_balancer"
 
-			if isSticky(host, location, backend.SessionAffinity.CookieSessionAffinity.Locations) {
-				upstreamName = fmt.Sprintf("sticky-%v", upstreamName)
-			}
+	if !dynamicConfigurationEnabled {
```

Comment:
This is currently buggy, it will ignore `https` proto when dynamic configuration is enabled. I have fixed it at https://github.com/Shopify/ingress/pull/29/files. @aledbf let me know if you want me include the fix in this PR or in a separate PR after this gets merged.

Reply:
Please include the fix in this PR